### DISCUSSION TITLE
Nuget Feed for stable/0.10.x

### DISCUSSION THREAD

**Discussion by workgroupengineering** [AvaloniaUI/Avalonia#7942] [Q&A] [answered]
Is there a nuget feed for the stable/0.10.x branch?

**Comment by maxkatz6** [comment, marked as answer]
It should be part of https://nuget.avaloniaui.net/repository/avalonia-nightly/index.json feed. Can't check right now, but I used it before.

**Reply by workgroupengineering** [reply]
Hi,
i had seen this feed before but it contains all master builds. I would only need the ones from the stable branch so that I can generate a github workflow that checks for new builds that have not yet been released on nuget.org to verify that the linked library is compatible or does not give compile errors.

eg: FluentAvalonia 1.3.2 is not compatible with 0.10.999-cibuild0019213-beta
